index.js

```diff
@@ -44,7 +44,7 @@ function init(
         port: configWithDefaults.websocketPort,
     });
 
-    setupKeyInput(screen.driver);
+    if(process.stdin.setRawMode) setupKeyInput(screen.driver);
```

Comment:
It might be a bit nicer to do this check in `setupKeyInput` instead? Something like:
```javascript
function setupKeyInput(driver) {
    if(!process.stdin.setRawMode) {
      return;
    }
    readline.emitKeypressEvents(process.stdin);
    process.stdin.setRawMode(true);
    process.stdin.addListener('keypress', (key, data) => {
        if (data.ctrl && data.name === 'c') {
            driver.sleep();
            process.exit();
        }
    });
}
```